src/blocking/rng.rs

```diff
@@ -0,0 +1,10 @@
+//! Blocking hardware random number generator
+
+/// Blocking read
+pub trait Read {
```

Comment:
This trait should be gated behind `#[cfg(feature = "unproven")]`, in addition to the gate in `prelude`.

At least, that's what I gather from the crate docs and the other unproven traits (like `digital::InputPin`)

Reply:
That can certainly be done. However the rules say that a trait is proven if at least two implementations and a driver or demo application exists and that is actually already the case. 😉